vnext/JSI/Shared/JSI.Shared.vcxitems.filters

```diff
@@ -21,10 +21,14 @@
     <ClInclude Include="$(MSBuildThisFileDirectory)ScriptStore.h">
       <Filter>Header Files</Filter>
     </ClInclude>
+    <ClInclude Include="$(MSBuildThisFileDirectory)ChakraRuntimeArgs.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)ChakraRuntimeFactory.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)ChakraRuntime.h" />
```

Comment:
><ClInclude Include="$(MSBuildThisFileDirectory)ChakraRuntime.h" /> [](start = 4, length = 66)

nit: Can we add Filters here so thing show up nicely in visual studio?

Reply:
Can do. This as is was just from Visual Studio automatically populating.